packages/serviceplaceholder-adapter/src/definitions/references.ts

```diff
@@ -51,6 +63,9 @@ const REFERENCE_RULES: referenceUtils.FieldReferenceDefinition<never>[] = [
   },
 ]
 
-export const REFERENCES: definitions.ApiDefinitions['references'] = {
+export const REFERENCES: definitions.ApiDefinitions<Options>['references'] = {
   rules: REFERENCE_RULES,
+  contextStrategyLookup: {
+    useParentType: ({ instance }) => _.get(instance.value, 'parent_type'),
```

Comment:
nit: for consistency with the existing ones 
```suggestion
    parentType: ({ instance }) => _.get(instance.value, 'parent_type'),
```